<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="videoLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>1011</width>
     <height>621</height>
    </rect>
   </property>
   <property name="text">
    <string>view</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fingercount">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>680</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>fingercount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>680</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="virtualMouse">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>680</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>mouse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
